conda-recipe/meta.yaml

```diff
@@ -19,6 +19,7 @@ requirements:
     - python >=3.5
     - requests
     - krtc
+    - ophyd
```

Comment:
I sat there for a while as well.  I think to remove the ophyd dependency requires removing the functionality.  As I think about this more, maybe it's a good enough reason to just make a plan stub that posts a message rather than hijacking an existing message.